_snippets/self-hosting/installation/latest-next-version.md

```diff
@@ -2,5 +2,5 @@
 n8n releases a new minor version most weeks. The `latest` version is for production use. `next` is the most recent release. You should treat `next` as a beta: it may be unstable. To report issues, use the [forum](https://community.n8n.io/c/questions/12){:target=_blank .external-link}.
 
 Current `latest`: 1.54.4  
```

Comment:
shouldn't this change too?

Reply:
This is automated, Looks like npm is showing latest as being anything other than 1.54.4 at the moment.